Makefile

```
@@ -317,6 +317,11 @@ test-yaml:
 	@# Fall back to find if there is no git, e.g. in package builds
 	yamllint --strict $$((git ls-files "*.yml" "*.yaml" 2>/dev/null || find -name '*.y*ml') | grep -v ^dbicdh)
 
+.PHONY: test-shfmt
+test-shfmt:
+	@which shfmt >/dev/null 2>&1 || (echo "Command 'shfmt' not found, can not execute bash script syntax checks" && false)
+	shfmt -d .
```

I suggest to do the following right now:
```suggestion
	shfmt -w -i 4 -bn -ci -sr $(shfmt -f .)
```
and remove the .editorconfig symlink.
We can think of any improvements later and get this big PR merged.

---

Considering we were not able to make eslint/prettier ignore the editor config (after reading https://github.com/prettier/eslint-plugin-prettier, https://prettier.io/docs/en/configuration.html, https://prettier.io/docs/en/options.html and https://eslint.org/docs/latest/use/configure/configuration-files#using-configuration-files and trying several variants) that might be the best way to move this forward for now, indeed.

---

And apparently 
```
shfmt -d -i 4 -bn -ci -sr .
```
also works. Not sure why we ruled that out when we experimented with it in the scripts repo.

---

oh right, because of the `.bpan` directory which we wanted to ignore.
So we might need something like this, depending on the repo:
```
shfmt -d -i 4 -bn -ci -sr  $(shfmt -f . | grep -v '^.bpan' )
```